src/normalize/entities.ts

```diff
@@ -119,6 +124,12 @@ const InviteComponent = new Entity({
 	seq: 'sequence'
 });
 
+const InviteReplies = new Entity({
+	...commonMessageFields,
```

Comment:
InviteReplies has only one key, so I guess these two lines are not required

Reply:
Done